build.rs

```diff
@@ -150,18 +250,21 @@ fn main() {
 
     // Second, we use bindgen to generate the Rust types
 
-    let bindings_rs = PathBuf::from(env::var("OUT_DIR").unwrap())
-        .join("bindings.rs");
+    let bindings_rs = PathBuf::from(env::var("OUT_DIR").unwrap()).join("bindings.rs");
```

Comment:
May you please refrain from such cosmetic changes which add noise to the diff?  I understand that one may want to run `rustfmt` on the file but then it is better IMHO to make it a separate PR.

Reply:
apologies, I typically have autoformat on in my IDE, so I didn't realise I was making these changes.

Reply:
removed now